src/_utils/utilities.js

```diff
@@ -10,11 +10,9 @@ import { getComponents } from './getComponents.js';
  * @param {string} propertyPrefix - Property for the css value to be created. Postfix will be appended according to direction matched.
  * @see {@link directionMap}
  */
-// TODO: hook this into the 'bounding' utils
 export function directionSize(propertyPrefix) {
   return ([_, direction, size], { theme }) => {
-    const v = theme.spacing?.[size] ?? h.global.auto(size);
-    if (!v) return warnOnce(`${propertyPrefix} not available in size ${size}`);
+    const v = theme.spacing?.[size] ?? h.bracket.global.auto.fraction(size);
```

Comment:
Does it mean that a classname `m-13` will be accepted? Or would it need to be `m-[13]`?